node/commons/compatibility-layer.ts

```diff
@@ -232,6 +234,11 @@ const fillSearchItemWithSimulation = (searchItem: SearchItem, orderFormItems: Or
       seller.commertialOffer.PriceValidUntil = priceValidUntil
       seller.commertialOffer.ListPrice = listPrice / 100
       seller.commertialOffer.PriceWithoutDiscount = price / 100
+      seller.commertialOffer.Teasers = getTeasersFromRatesAndBenefitsData(orderFormItem.ratesAndBenefitsData)
+      seller.commertialOffer.DiscountHighLight = getDiscountHighLightsFromRatesAndBenefitsData(orderFormItem.ratesAndBenefitsData)
```

Comment:
what do you think of the names being just `getTeasers` and `getDiscountHighlights` or something like that?

Reply:
Done! 🚀 
I also added those types that you asked 😉